docs/python_docs/python/tutorials/performance/backend/mkldnn/index.rst

```diff
@@ -32,9 +32,15 @@ Intel MKL-DNN
 
       How to perform quantization with MKLDNN
 
+   .. card::
+      :title: Connectionist Temporal Classification
+      :link: speech_recognition/ctc.html
```

Comment:
that has nothing to do with mkldnn

Reply:
That's a weird merge artifact... I'll remove it.